### PR TITLE
SCRUM-2833: add genetic modifiers

### PR DESCRIPTION
**Pull request by abecerra**

### CODE REVIEW

File: apps/main-app/src/components/dataTable/AnnotatedEntitiesPopupCuration.js [hunk 1]
```diff
@@ -66,6 +67,7 @@ function AnnotatedEntitiesPopupCuration(props) {
                 <th>Additional implicated genes</th>
                 <th>Experimental condition</th>
                 <th>Modifier</th>
+                <th>Genetic Modifier</th>
```
Comment (markquintontulloch) [inline apps/main-app/src/components/dataTable/AnnotatedEntitiesPopupCuration.js:70]: Should be pluralised?

Comment (abecerra) [reply apps/main-app/src/components/dataTable/AnnotatedEntitiesPopupCuration.js]: Thank you, pluralized in last commit!

**Comment by markquintontulloch** [comment]
Looks okay to me, but will let Adam / Olin review as I'm not particularly familiar with the main site UI

File: apps/main-app/src/components/dataTable/GeneticModifiersCellCuration.js [hunk 2]
```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import { CollapsibleList } from '../collapsibleList';
+import ExternalLink from '../ExternalLink';
+import { getResourceUrl } from './getResourceUrl';
+
+
+function GeneticModifier(modifier) {
+    let url = getResourceUrl(modifier.curie, modifier.type);
```
[the remark this answers is not in the document]
Comment (abecerra) [reply apps/main-app/src/components/dataTable/GeneticModifiersCellCuration.js:8]: I'll ask!

[the remark this answers is not in the document]
Comment (abecerra) [reply apps/main-app/src/components/dataTable/GeneticModifiersCellCuration.js]: passing subtype, getResourceURL handles the case where it is undefined

**Review by adamgibs — COMMENTED** [review]
Looks good to me for the most part. One question: does the pathway widget code need to be modified?

**Review by adamgibs — APPROVED** [review]

**Pull request merged**